test/specs/utils/tablify.js

```diff
@@ -42,6 +42,14 @@ describe('tablify util', () => {
 			it('Then the returned table should have a row with the object’s nested values', then.theReturnedTableShouldHaveAHeadWithTheObjectNestedValues);
 		});
 	});
+	describe('Given a deeper nested object', () => {
+		beforeEach(given.aDeeperNestedObject);
+		describe('When the object is tablified', () => {
+			beforeEach(when.theObjectIsTablified);
+			it('Then the returned table should have a head with the object’s nested keys', then.theReturnedTableShouldHaveAHeadWithTheObjectDeepNestedKeys);
+			it('Then the returned table should have a row with the object’s nested values', then.theReturnedTableShouldHaveAHeadWithTheObjectDeepNestedValues);
```

Comment:
Descriptions diverge from function names. Should be `Then the returned table should have a head with the object’s deeply nested keys/values`.